docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -22,18 +22,34 @@ The procedure described below uses a built-in questionnaire for assessing the ri
 +
 [NOTE]
 ====
-Only one application can be assessed at a time.
+You can assess only one application at a time.
 ====
-. Click  *Assess*.
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
+. Select an application from the list that appears in the main pane, click the Options menu {kebab} at the right end of the row, and select *Assess* from the drop-down menu.
 +
-// ![](/Tackle2/AppAssessAnalyze/AssessDetails.png)
+// Get updated image for MTA
```

Comment:
You have not checked-out correctly so overwritten corrections with older versions

You need to learn Git basics